testing/rscsrvtest/setup.go

```diff
@@ -6,14 +6,69 @@ import (
 	"github.com/onsi/gomega"
 )
 
-func Setup(services ...rscsrv.Service) {
+type Setupper interface {
+	Setup(...rscsrv.Service)
+	Before(func()) Setupper
+	After(func()) Setupper
+}
+
+type starter struct {
+	offset int
+	before []func()
+	after  []func()
+}
+
+func (s *starter) Before(fn func()) Setupper {
+	if s.before == nil {
+		s.before = make([]func(), 0, 5)
```

Comment:
Why 5?

Reply:
I don't know... just to have some allocated already.